2023-05-07 00:20:23,935 INFO    MainThread:14673 [wandb_setup.py:_flush():76] Configure stats pid to 14673
2023-05-07 00:20:23,936 INFO    MainThread:14673 [wandb_setup.py:_flush():76] Loading settings from /home/joowan/.config/wandb/settings
2023-05-07 00:20:23,936 INFO    MainThread:14673 [wandb_setup.py:_flush():76] Loading settings from /home/joowan/Desktop/face_pr/wandb/settings
2023-05-07 00:20:23,936 INFO    MainThread:14673 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-05-07 00:20:23,936 INFO    MainThread:14673 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-05-07 00:20:23,936 INFO    MainThread:14673 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-05-07 00:20:23,936 INFO    MainThread:14673 [wandb_init.py:_log_setup():507] Logging user logs to /home/joowan/Desktop/face_pr/wandb/run-20230507_002023-rk9pgyn1/logs/debug.log
2023-05-07 00:20:23,936 INFO    MainThread:14673 [wandb_init.py:_log_setup():508] Logging internal logs to /home/joowan/Desktop/face_pr/wandb/run-20230507_002023-rk9pgyn1/logs/debug-internal.log
2023-05-07 00:20:23,936 INFO    MainThread:14673 [wandb_init.py:init():547] calling init triggers
2023-05-07 00:20:23,936 INFO    MainThread:14673 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-05-07 00:20:23,936 INFO    MainThread:14673 [wandb_init.py:init():571] re-initializing run, found existing run on stack: mk2d25ju
2023-05-07 00:20:23,937 INFO    MainThread:14673 [wandb_run.py:_finish():1892] finishing run kookmin_ai/single tasks result/mk2d25ju
2023-05-07 00:20:23,938 ERROR   MainThread:14673 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2023-05-07 00:20:23,938 INFO    MainThread:14673 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-05-07 00:20:23,938 INFO    MainThread:14673 [wandb_init.py:_jupyter_teardown():435] cleaning up jupyter logic
2023-05-07 00:20:23,938 INFO    MainThread:14673 [wandb_run.py:_atexit_cleanup():2126] got exitcode: 0
2023-05-07 00:20:23,938 INFO    MainThread:14673 [wandb_run.py:_restore():2109] restore
2023-05-07 00:20:23,938 INFO    MainThread:14673 [wandb_run.py:_restore():2115] restore done
2023-05-07 00:20:38,363 INFO    MainThread:14673 [wandb_init.py:init():595] starting backend
2023-05-07 00:20:38,363 INFO    MainThread:14673 [wandb_init.py:init():599] setting up manager
2023-05-07 00:20:38,368 INFO    MainThread:14673 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-05-07 00:20:38,373 INFO    MainThread:14673 [wandb_init.py:init():605] backend started and connected
2023-05-07 00:20:38,380 INFO    MainThread:14673 [wandb_run.py:_label_probe_notebook():1237] probe notebook
2023-05-07 00:20:38,380 INFO    MainThread:14673 [wandb_run.py:_label_probe_notebook():1247] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-05-07 00:20:38,380 INFO    MainThread:14673 [wandb_init.py:init():695] updated telemetry
2023-05-07 00:20:38,712 INFO    MainThread:14673 [wandb_init.py:init():732] communicating run to backend with 60.0 second timeout
